test/datasource.test.js

```diff
@@ -129,7 +129,55 @@ describe('loopback:datasource generator', function() {
     });
   });
 
-  it('should support IBM Object Storage ', function(done) {
+  it('should support MongoDB', function(done) {
+    var datasourceGen = givenDataSourceGenerator();
+    helpers.mockPrompt(datasourceGen, {
+      name: 'mongodb-datasource',
+      connector: 'mongodb',
+      installConnector: false,
+    });
+
+    datasourceGen.run(function() {
+      var datasources = readDataSourcesJsonSync('server');
+      // eslint-disable-next-line no-unused-expressions
+      expect(datasources['mongodb-datasource']).to.exist;
```

Comment:
https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access

Ideally, generator-loopback should configure https://github.com/prodatakey/dirty-chai to enable method-based assertions.

Since that's out of scope of this patch, I am proposing to assert the type of the datasource config instead:

```js
expect(datasources['mongodb-datasource']).to.be.an('object');
```